dependencies.yml

```diff
@@ -1,3 +1,5 @@
+- src: https://github.com/coopdevs/certbot_nginx
+  version: feature/separate-cert-creation-to-task
```

Comment:
Specify the version once https://github.com/coopdevs/certbot_nginx/pull/10 get merged and released.